core/components/product-card/index.tsx

```diff
@@ -48,6 +53,16 @@ export const ProductCard = ({
       price={price}
       showCompare={showCompare}
       subtitle={brand?.name}
+      wishlistSheet={
+        showWishlistSheet &&
+        wishlistsList && (
```

Comment:
do we need to check `wishlistsList.length > 0` here ?

Reply:
yes, because be we still need to be sure that wishlistsList was fetched and provided from upper scope

Reply:
no, because we always will have favourites wishlist, so we only need to be sure that `wishlistsList` is not `undefined`